.github/workflows/run_tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
at this point, according to https://endoflife.date/python, 3.5 and 3.6 can be dropped too, though we may want to drop that later on